scripts/netobserv.sh

```diff
@@ -106,6 +123,33 @@ patch_netobserv() {
     return 1
   fi
 }
+
+patch_unreleased_images(){
+  if [[ $INSTALLATION_SOURCE == "Internal" ]]; then
+    QUAY_URL="quay.io/redhat-user-workloads/ocp-network-observab-tenant"
+
+    # capture stream from quay.io/redhat-user-workloads/ocp-network-observab-tenant/catalog-zstream:latest
+    STREAM=${DOWNSTREAM_IMAGE%%:*}
+    STREAM=${STREAM##*-}
+    CSV=$(oc get csv -n openshift-netobserv-operator | grep -iE "net.*observ" | awk '{print $1}')
+
+    # capture sha256 of images
+    RELATED_IMAGE_EBPF_AGENT=$(oc -n openshift-netobserv-operator get csv/$CSV -o jsonpath='{.spec.install.spec.deployments[0].spec.template.spec.containers[0].env[?(@.name=="RELATED_IMAGE_EBPF_AGENT")].value}')
+    RELATED_IMAGE_EBPF_AGENT=${RELATED_IMAGE_EBPF_AGENT#*@}
+    RELATED_IMAGE_FLOWLOGS_PIPELINE=$(oc -n openshift-netobserv-operator get csv/$CSV -o jsonpath='{.spec.install.spec.deployments[0].spec.template.spec.containers[0].env[?(@.name=="RELATED_IMAGE_FLOWLOGS_PIPELINE")].value}')
+    RELATED_IMAGE_FLOWLOGS_PIPELINE=${RELATED_IMAGE_FLOWLOGS_PIPELINE#*@}
+    RELATED_IMAGE_CONSOLE_PLUGIN=$(oc -n openshift-netobserv-operator get csv/$CSV -o jsonpath='{.spec.install.spec.deployments[0].spec.template.spec.containers[0].env[?(@.name=="RELATED_IMAGE_CONSOLE_PLUGIN")].value}')
+    RELATED_IMAGE_FLOWLOGS_PIPELINE=${RELATED_IMAGE_CONSOLE_PLUGIN#*@}
```

Comment:
```suggestion
    RELATED_IMAGE_CONSOLE_PLUGIN=${RELATED_IMAGE_CONSOLE_PLUGIN#*@}
```
my bad, missed this one in last review